frontend/src/static/js/v_zoom.js

```diff
@@ -7,6 +9,9 @@ window.vZoom = {
       expandedCommitMessagesCount: this.totalCommitMessageBodyCount,
       commitsSortType: 'time',
       toReverseSortedCommits: true,
+      shouldShowFileTypes: true,
```

Comment:
I feel we dont really need this option, can make it as the same as authorship one.